Vehicle.cs

```diff
@@ -0,0 +1,72 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CodeReviewDemo
+{
+    internal class Vehicle : IVehicle
+    {
+        // Manufacturer of the vehicle
+        public string Manufacturer { get; set; }
+
+        // Model of the vehicle
```

Comment:
It's better to use PascalCase for public properties in C#. The property `model` should be renamed to `Model` and `year` to `Year` to follow the standard naming conventions.